Resources/Prototypes/_LostParadise/Roles/play_time_trackers.yml

```
@@ -14,4 +14,4 @@
   id: JobPilot
 
 - type: playTimeTracker
-  id: LPPJobSecurityEnginer
\ No newline at end of file
+  id: LPPJobSecurityEngineer
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)